Cargo.toml

```diff
@@ -1,3 +1,5 @@
+lints.workspace = true
+
```

Comment:
issue: It seems to be placed at a strange place

Reply:
I remember that it's the only place in this file where it works. I was struggling with setting it, but I found a project that was setting it at the beginning of the file, and it worked in sspi-rs as well.

Reply:
Done. Thank you